release/_publishing.sh

```diff
@@ -35,6 +35,30 @@ function init_gcs {
 	gcloud auth activate-service-account --key-file "${LIFERAY_RELEASE_GCS_TOKEN}"
 }
 
+function update_latest_in_bundles {
+    local file_path="${BASE_DIR}/bundles.yml"
+
+    if (yq eval ".quarterly | has(\"${_PRODUCT_VERSION}\")" "${file_path}" | grep -q "true")
+    then
+        lc_log INFO "The ${_PRODUCT_VERSION} product version was already published. Reuploading to Docker Hub."
+        return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
```

Comment:
@kiwm, please add a blank line between these lines.

Additionally, why do you say `Reuploading...` and then you exit the function? Can you please remove the second part of the message related to the reuploading?

Reply:
In this function, I am updating the bundles.yml file. If there is no need to update the bundles.yml (the necessary changes are already present), that means the Docker image is already published to Docker Hub. So I added the reupload part. But I will remove.